website/src/components/doc-page/doc-page-navigation.tsx

```diff
@@ -256,16 +256,17 @@ export const DocPageNavigation: FunctionComponent<DocPageNavigationProperties> =
                 title: item!.title!,
                 items: item!.items
                   ? item?.items
-                    .filter((item) => !!item)
-                    .map<Item>((item) => ({
-                      path: item!.path!,
-                      title: item!.title!,
-                    }))
+                      .filter((item) => !!item)
+                      .map<Item>((item) => ({
+                        path: item!.path!,
+                        title: item!.title!,
+                      }))
                   : undefined,
               })),
-            `/docs/${activeProduct!.path!}${activeVersion?.path?.length && activeVersion.path.length > 0
-              ? "/" + activeVersion.path!
-              : ""
+            `/docs/${activeProduct!.path!}${
+              activeVersion?.path?.length && activeVersion.path.length > 0
```

Comment:
Use `!!activeVersion?.path?.length`instead of `activeVersion?.path?.length && activeVersion.path.length > 0`.

Reply:
I now that you just reformatted that, but when touching then also improve 😄

Reply:
I think these changes come from running prettiert :D?